tests/cpu_bugs/ipmi_to_qemu.pm

```diff
@@ -0,0 +1,97 @@
+        zypper_call("ar http://download.opensuse.org/repositories/devel:/openQA:/SLE-15/SLE_15/devel:openQA:SLE-15.repo");
+    } elsif (is_sle(">=12")) {
+        zypper_call("ar http://download.opensuse.org/repositories/devel:/languages:/perl/SLE_12_SP4/devel:languages:perl.repo");
+        zypper_call("ar http://download.opensuse.org/repositories/devel:/openQA/SLE_12_SP4/devel:openQA.repo");
+        zypper_call("ar http://download.opensuse.org/repositories/devel:/openQA:/SLE-12/SLE_12_SP4/devel:openQA:SLE-12.repo");
+    }
+    script_run("zypper -n --gpg-auto-import-keys ref");
+    script_run("zypper -n dup --auto-agree-with-licenses");
+    zypper_call("in openQA-worker perl-DBIx-Class-DeploymentHandler perl-YAML-Tiny");
+    zypper_call("in --replacefiles perl-DBD-SQLite");
+
+    #NFS mount
+    #assert_script_run("mount -t nfs $nfs_hostname:/var/lib/openqa/share /var/lib/openqa/share");
+
+    #Rsync
+    if (script_run("grep \"^\\[http://$nfs_hostname\\]\" /etc/openqa/workers.ini") eq 1) {
+        assert_script_run("echo \"[http://$nfs_hostname]\" >> /etc/openqa/workers.ini");
+        assert_script_run("echo \"TESTPOOLSERVER = rsync://$nfs_hostname/tests\" >>/etc/openqa/workers.ini");
+    }
+    if (script_run("grep \"^CACHEDIRECTORY = /var/lib/openqa/cache\" /etc/openqa/workers.ini") eq 1) {
+        assert_script_run("sed -i '/^#HOST.*=.*/a CACHELIMIT = 200' /etc/openqa/workers.ini");
+        assert_script_run("sed -i '/^#HOST.*=.*/a CACHEDIRECTORY = /var/lib/openqa/cache' /etc/openqa/workers.ini");
+    }
+
+    assert_script_run("sed -i '/^#.*global/s/^#//' /etc/openqa/workers.ini");
+    assert_script_run("sed -i '/^HOST =.*/d' /etc/openqa/workers.ini");
+
+
+    if (script_run("grep \"^#HOST.*=.*\" /etc/openqa/workers.ini") == 0) {
+        assert_script_run("sed -i '/^#HOST.*=.*/a HOST = $webui_hostname' /etc/openqa/workers.ini");
+    } else {
+        assert_script_run("sed -i '/^\\[global\\]/a HOST = $webui_hostname' /etc/openqa/workers.ini");
+    }
+    assert_script_run("sed -i '/^WORKER_CLASS =.*/d' /etc/openqa/workers.ini");
+    assert_script_run("sed -i '/WORKER_HOSTNAME =.*/a WORKER_CLASS = $qemu_worker_class' /etc/openqa/workers.ini");
+    assert_script_run(
+        'curl '
+          . $cliect_ini_url
+          . ' -o /etc/openqa/client.conf',
+        60
+    );
+    script_run('systemctl start openqa-worker-cacheservice-minion');
+    script_run('systemctl start openqa-worker-cacheservice');
+    script_run('systemctl start openqa-worker@1');
+    script_run('systemctl start openqa-worker@2');
+    script_run('systemctl start openqa-worker@3');
+    script_run('systemctl start openqa-worker@4');
+    script_run('systemctl start openqa-worker@5');
+    script_run('systemctl start openqa-worker@6');
+    script_run('systemctl start openqa-worker@7');
+    script_run('systemctl start openqa-worker@8');
+}
+
+sub test_flags {
+    return {milestone => 1, fatal => 0};
+}
```

Comment:
I don't think you need to overwrite fatal, shouldn't tests be non-fatal by default?

Reply:
OK. I could remove it.